dist-git/Dockerfile

```diff
@@ -0,0 +1,133 @@
+
+# Build WMCO
+WORKDIR $REMOTE_SOURCE_DIR/build/windows-machine-config-operator
+# Copy .git metadata so that we can generate the version for the WMCO binary
+COPY $REMOTE_SOURCE/app/.git .git
+# Copy files and directories needed to build the WMCO binary
+COPY $REMOTE_SOURCE/app/build build
+COPY $REMOTE_SOURCE/app/cmd cmd
+COPY $REMOTE_SOURCE/app/deploy deploy
+COPY $REMOTE_SOURCE/app/hack hack
+COPY $REMOTE_SOURCE/app/pkg pkg
+COPY $REMOTE_SOURCE/app/test test
+COPY $REMOTE_SOURCE/app/vendor vendor
+COPY $REMOTE_SOURCE/app/version version
+COPY $REMOTE_SOURCE/app/go.mod go.mod
+COPY $REMOTE_SOURCE/app/go.sum go.sum
+COPY $REMOTE_SOURCE/app/Makefile Makefile
+COPY $REMOTE_SOURCE/app/tools.go tools.go
+COPY $REMOTE_SOURCE/app/bundle.Dockerfile bundle.Dockerfile
+RUN make build
+
+# Build WMCB
+WORKDIR $REMOTE_SOURCE_DIR/build/windows-machine-config-operator/windows-machine-config-bootstrapper/
+COPY $REMOTE_SOURCE/app/windows-machine-config-bootstrapper/ .
+RUN make build
+
+# Build hybrid-overlay
+WORKDIR $REMOTE_SOURCE_DIR/build/windows-machine-config-operator/ovn-kubernetes/
+COPY $REMOTE_SOURCE/app/ovn-kubernetes/ .
+WORKDIR $REMOTE_SOURCE_DIR/build/windows-machine-config-operator/ovn-kubernetes/go-controller/
+RUN make windows
+
+# Build Kubernetes node binaries
+WORKDIR $REMOTE_SOURCE_DIR/build/windows-machine-config-operator/kubernetes/
+COPY $REMOTE_SOURCE/app/kubernetes/ .
+ENV KUBE_BUILD_PLATFORMS windows/amd64
+RUN make WHAT=cmd/kubelet
+RUN make WHAT=cmd/kube-proxy
+
+# Build CNI plugins
+WORKDIR $REMOTE_SOURCE_DIR/build/windows-machine-config-operator/containernetworking-plugins/
+COPY $REMOTE_SOURCE/app/containernetworking-plugins/ .
+ENV CGO_ENABLED=0
+RUN ./build_windows.sh
+
+# Build the operator image with following payload structure
+# /payload/
+#├── cni
+#│   ├── flannel.exe
+#│   ├── host-local.exe
+#│   ├── win-bridge.exe
+#│   ├── win-overlay.exe
+#│   └── cni-conf-template.json
+#├── hybrid-overlay-node.exe
+#├── kube-node
+#│   ├── kubelet.exe
+#│   └── kube-proxy.exe
+#├── powershell
+#│   └── wget-ignore-cert.ps1
+#│   └── hns.psm1
+#└── wmcb.exe
+
+FROM openshift/golang-builder:rhel_8_1.15
+LABEL stage=operator
+# This label maps to the brew build target
+LABEL com.redhat.component="windows-machine-config-operator-container" \
+    name="openshift4/windows-machine-config-operator" \
+    summary="OpenShift Windows Machine Config Operator" \
+    License="ASL 2.0" \
+    io.k8s.display-name="OpenShift Windows Machine Config Operator" \
+    io.k8s.description="OpenShift Windows Machine Config Operator" \
+    io.openshift.tags="openshift,windows" \
+    io.openshift.maintainer.product="OpenShift Container Platform" \
+    io.openshift.maintainer.component="Windows Containers" \
+    com.redhat.delivery.appregistry="false" \
+    version="0.0.1" \
+    maintainer="team-winc@redhat.com"
+
+# Install tar so that we do "oc cp" to extract the manifests in the CPaaS pipeline
+RUN yum -y install tar
```

Comment:
This will become `microdnf` if we move to `ubi-minimal`. Please add `&& microdnf clean`.